diesel/src/query_builder/bind_collector.rs

```diff
@@ -125,6 +142,26 @@ where
     }
 }
 
+impl<DB> MoveableBindCollector<DB> for RawBytesBindCollector<DB>
+where
+    for<'a> DB: Backend<BindCollector<'a> = Self> + TypeMetadata + 'static,
+    <DB as TypeMetadata>::TypeMetadata: Clone + Send,
+{
+    type BindData = Self;
+
+    fn moveable(&self) -> Self::BindData {
+        RawBytesBindCollector {
+            binds: self.binds.clone(),
+            metadata: self.metadata.clone(),
+        }
+    }
+
+    fn append_bind_data(&mut self, from: &Self::BindData) {
+        self.binds = from.binds.clone();
+        self.metadata = from.metadata.clone();
```

Comment:
```suggestion
        self.binds.extend(from.binds.cloned());
        self.metadata.extend(from.metadata.clone());
```

To make this behave similar than the sqlite implementation

Reply:
Done, just minor change:

```
self.binds.extend(from.binds.iter().cloned())
```